analytics/tracker.go

```diff
@@ -373,7 +373,7 @@ func (t tracker) SendToolSetupEvent(
 		"tool_name":            request.ToolName,
 		"requested_version":    request.UnparsedVersion,
 		"resolution_strategy":  resolutionStrategy,
-		"custom_plugin_id":     request.PluginIdentifier,
+		"custom_plugin_id":     request.PluginURL,
```

Comment:
I think it's fine and we can live with it, it would be a lot of hassle to migrate every consumer of this data to a renamed field.